ytgov/elcc-data-management
# Cleanup: Drop Now Unused Funding Submission Line Value Table and Model

# Context

Clean up the unusesd FundingSubmissionLineValue table and model.

# Testing Instructions

1. Boot the app via `dev up`
2. Check that the app complies, and that you can log in at http://localhost:8080.
